frontend/components/selector/SelectorTopBar.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div v-if="selectors.length > 0" class="w-full h-fit">
+    <div class="flex flex-row"> 
+      <button
+        v-for="t in selectors"
+        class="w-full justify-center bg-light-header dark:bg-dark-header flex px-3 py-1 border-l-[1px] border-y-[1px] border-light-text dark:border-dark-text text-light-text dark:text-dark-text hover:bg-black/10 dark:hover:bg-white/30"
+      >

```

Comment:
@sayik, one think I'm realizing is that we need to add styling for whichever option is selected. Do you want to work on this, or should I? We'd need another prop like say `selected` that's a string, and then we can do a check for if `s === selected` and change the styling if this is the case. How does this sound?

Reply:
For this we'd do a conditional `:class` like the following:

```
:class="{
  'bg-ETC': s === selected,
  'bg-ETC': s !== selected
}"
```

You can find examples of how to do this in many places in the code base by searching for `:class="` :)